cfbs.json

```diff
@@ -118,6 +118,45 @@
         "copy policy/default-encrypt-method-sha512.cf services/security-hardening/default-encrypt-method-sha512/default-encrypt-method-sha512.cf"
       ]
     },
+    "delete-files": {
+      "description": "Manages file permissions",
+      "tags": ["management"],
+      "repo": "https://github.com/nickanderson/cfengine-delete-files",
+      "by": "https://github.com/nickanderson",
+      "version": "0.0.1",
+      "commit": "f61a3a105f6e15b5b0fcf7649d725d1d3112a64d",
+      "steps": [
+        "copy delete-files.cf services/cfbs/modules/delete-files/delete-files.cf",
+        "input delete-files/input.json def.json",
+        "bundles delete_files:delete_files",
+        "policy_files services/cfbs/modules/delete-files/delete-files.cf"
+      ],
+      "input": [
+        {
+          "type": "list",
+          "variable": "files",
+          "namespace": "delete_files",
+          "bundle": "delete_files",
+          "label": "Files",
+          "subtype": [
+            {
+              "key": "path",
+              "type": "string",
+              "label": "Path",
+              "question": "Path to file"
+            },
+            {
+              "key": "why",
+              "type": "string",
+              "label": "Why",
+              "question": "Why does this file absence matter?",
+              "default": "Unknown"
```

Comment:
Yea, maybe. We merged the other module for file permissions in this way, so it also makes sense to keep them the same (maybe change both to make Why not optional?).

Reply:
How do you make it optional?

Reply:
AFAIK optional variables are not supported (yet). But I imagine you can implement it in policy? If variable is empty string; do something else, etc

Reply:
There isn't really anything to DO here. If the variable is not defined then the whole promise will be skipped (which would be a problem). If the variable is an empty string, then it's just an empty string. So, the comment would just be an empty string.

Reply:
One thing we could do is force the user (using cfbs or MP) to put something there instead of pressing enter / empty string. But that's outside the scope of this PR, requires cfbs changes.